src/main/java/com/bitquest/bitquest/BitQuest.java

```diff
@@ -811,6 +811,8 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
                     } catch (IOException e1) {
                         e1.printStackTrace();
                     }
+                    if (sendAmount >= 20000)
+                    {
                     try {
```

Comment:
Im sorry, I wasn't trying to be rude (also: I literally did this on my iPhone, so it wasn't really easy to indent it all)